src/common/config/client/db/configdriver.go

```diff
@@ -0,0 +1,105 @@
+package db
+
+import (
+	"sync"
+
+	"github.com/goharbor/harbor/src/common/config/encrypt"
+
+	"github.com/goharbor/harbor/src/common/config"
+	"github.com/goharbor/harbor/src/common/dao"
+	"github.com/goharbor/harbor/src/common/models"
+	"github.com/goharbor/harbor/src/common/utils/log"
+)
+
+// ConfigureDriver - Retrieve configurations from database
+type ConfigureDriver struct {
```

Comment:
Similarly as the `config` is in the package path it should probably be called `Driver`